docs/platforms/javascript/guides/nuxt/install/dynamic-import.mdx

```diff
@@ -6,37 +6,17 @@ description: "Learn about how the Nuxt SDK leverages dynamic input() in the buil
 
 ## Understanding the `import()` expression
 
-The `import()` expression (also called "dynamic import") allows conditional and flexible module loading in ESM.
-For the Sentry Nuxt SDK, it provides an approach to initialize server-side configuration before application startup.
+<Alert level='warning'>
+  This installation method does not work with more recent versions of Nuxt/Nitro.
 
-The Sentry Nuxt SDK will be initialized before any other code runs and the Nitro server runtime code will be loaded later because of `import()`ing it.
-This early initialization is required to set up the SDK's instrumentation of various libraries correctly.
-
-## Initializing Sentry with Dynamic `import()`
+  We recommend the guide for installing the SDK with the <PlatformLink to="/install/cli-import">CLI flag `--import`</PlatformLink> or <PlatformLink to="/install/limited-server-tracing">limited server tracing</PlatformLink>
+</Alert>
 
-Enable the dynamic `import()` by setting `autoInjectServerSentry`:
-
-```typescript {filename:nuxt.config.ts} {3}
-export default defineNuxtConfig({
-  sentry: {
-    autoInjectServerSentry: 'experimental_dynamic-import'
-  },
-})
-```
+The `import()` expression, or dynamic import, enables flexible, conditional module loading in ESM.
+For the Sentry Nuxt SDK, it provides an approach to initialize server-side Sentry configuration before application startup.
 
-After setting this, the Sentry Nuxt SDK adds some build-time configuration to ensure your app is wrapped with `import()`.
-With this, Sentry can be initialized before all other modules of the application.
-
-The Nuxt server entry file will look something like this:
-
-```javascript {filename:.output/server/index.mjs}
-// Note: The file may have some imports and code, related to debug IDs
-Sentry.init({
-  dsn: "..."
-});
-
-import('./chunks/nitro/nitro.mjs').then(function (n) { return n.r; });
-```
+The Sentry Nuxt SDK will be initialized before any other code runs and the Nitro server runtime code will be loaded later because of `import()`ing it.
+This early initialization is required to set up the SDK's instrumentation of various libraries correctly.
```

Comment:
```suggestion
The Sentry Nuxt SDK initializes before any other code runs, while the Nitro server runtime loads later because it is `import()`ed. This early setup ensures that the SDK instruments various libraries correctly.
```